diff --git a/rust/bindings/bindings_helper.h b/rust/bindings/bindings_helper.h
index 84303bf221dd9..5e5c00c655cf 100644
--- a/rust/bindings/bindings_helper.h
+++ b/rust/bindings/bindings_helper.h
@@ -16,6 +16,7 @@
 #include <linux/jiffies.h>
 #include <linux/mdio.h>
 #include <linux/miscdevice.h>
+#include <linux/moduleparam.h>
 #include <linux/phy.h>
 #include <linux/refcount.h>
 #include <linux/sched.h>
