diff --git a/rust/bindings/bindings_helper.h b/rust/bindings/bindings_helper.h
index 8f3f8e7b16d5..330160f11b60 100644
--- a/rust/bindings/bindings_helper.h
+++ b/rust/bindings/bindings_helper.h
@@ -17,6 +17,7 @@
 #include <linux/firmware.h>
 #include <linux/jiffies.h>
 #include <linux/mdio.h>
+#include <linux/moduleparam.h>
 #include <linux/phy.h>
 #include <linux/refcount.h>
 #include <linux/sched.h>
